src/client/components/Modals.tsx

```diff
@@ -96,49 +112,51 @@ export const EventsModal: React.FC<eventProps> = ({open, setOpen, startDateRef,
-                  <Grid xs={4} item style={{textAlign: "center"}}>
-                      <TextField required label="Interviewers" onChange={onTextFieldUpdate(setInterviewers)}/>
-                  </Grid>
-                  <Grid xs={4} item style={{textAlign: "center"}}>
-                      <TextField required label="Candidates" onChange={onTextFieldUpdate(setCandidates)}/>
-                  </Grid>
-                  <Grid xs={4} item style={{textAlign: "center"}}>
-                      <TextField required value={startDate || ""} helperText="Start Date" type="date" InputLabelProps={{ shrink: true }} onChange={onTextFieldUpdate(setStartDate)}/>
-                  </Grid>
-                  <Grid xs={4} item style={{textAlign: "center"}}>
-                      <TextField required value={startTime || ""} helperText="Start Time" type="time" InputLabelProps={{ shrink: true }} onChange={onTextFieldUpdate(setStartTime)}/>
-                  </Grid>
-                  <Grid xs={4} item style={{textAlign: "center"}}>
-                      <TextField required label="Problem" onChange={onTextFieldUpdate(setProblem)}/>
-                  </Grid>
-                  <Grid xs={4} item style={{textAlign: "center"}}>
-                      <TextField required value={endDate || ""} helperText="End Date" type="date" InputLabelProps={{ shrink: true }} onChange={onTextFieldUpdate(setEndDate)}/>
-                  </Grid>
-                  <Grid xs={4} item style={{textAlign: "center"}}>
-                      <TextField required value={endTime || ""} helperText="End Time" type="time" InputLabelProps={{ shrink: true }} onChange={onTextFieldUpdate(setEndTime)}/>
-                  </Grid>
-                  <Grid xs={4} item style={{textAlign: "center"}}>
-                      <TextField required label="Language" onChange={onTextFieldUpdate(setLanguage)}/>
-                  </Grid>
+        <Paper style={{minWidth:"335px", maxWidth:"700px", margin:"auto", outline: "none", padding: "15px"}}>
+          <div style={{display: "flex", justifyContent: "space-between"}}>
+            <h3 style={{margin:"10px"}}>Create/Edit Event</h3>
+            <IconButton color="secondary" onClick={handleClose}>
+              <CloseIcon/>
+            </IconButton>
+          </div>
+          <form ref={formRef}>
+            <Grid container direction="row" justify="center" style={{maxWidth: '100%'}} spacing={2}>
+                <Grid xs={4} item style={{textAlign: "center"}}>
+                    <TextField required label="Event Title" onChange={onTextFieldUpdate(setEventTitle)}/>
+                </Grid>
+                <Grid xs={4} item style={{textAlign: "center"}}>
+                    <TextField required label="Interviewers" onChange={onTextFieldUpdate(setInterviewers)}/>
+                </Grid>
+                <Grid xs={4} item style={{textAlign: "center"}}>
+                    <TextField required label="Candidates" onChange={onTextFieldUpdate(setCandidates)}/>
```

Comment:
This applies to everywhere, but please use 4 space indentation:
![image](https://user-images.githubusercontent.com/52944725/100520695-25c81d00-316d-11eb-8a25-d04122e7805d.png)